### PR TITLE
Add `#on_weekday?` method to `Date`, `Time`, and `DateTime`.

### PR DESCRIPTION
**Pull request by vipulnsward**
r? @dhh

### CODE REVIEW

**Comment by dhh** [comment]
💪

**Comment by vijaydev** [comment]
@vipulnsward can you also update the AS guide please? http://edgeguides.rubyonrails.org/active_support_core_extensions.html#extensions-to-datetime

File: guides/source/5_0_release_notes.md [hunk 1]
```diff
@@ -705,7 +705,7 @@ Please refer to the [Changelog][active-support] for detailed changes.
 *   Changed the default test order from `:sorted` to `:random`.
     ([commit](https://github.com/rails/rails/commit/5f777e4b5ee2e3e8e6fd0e2a208ec2a4d25a960d))
 
-*   Added `#on_weekend?`, `#next_weekday`, `#prev_weekday` methods to `Date`,
+*   Added `#on_weekend?`, `#on_weekday?`, `#next_weekday`, `#prev_weekday` methods to `Date`,
     `Time`, and `DateTime`.
     ([Pull Request](https://github.com/rails/rails/pull/18335))
```
Comment (vijaydev) [inline guides/source/5_0_release_notes.md]: We can add this PR link (23687) in the release notes too. The PR 18335 doesn't add on_weekday.

Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: Will do.

On Monday 15 February 2016, Vijay Dev <notifications@github.com> wrote:

> In guides/source/5_0_release_notes.md
> <https://github.com/rails/rails/pull/23687#discussion_r52901426>:
>
> > @@ -705,7 +705,7 @@ Please refer to the [Changelog][active-support] for detailed changes.
> >  *   Changed the default test order from `:sorted` to `:random`.
> >      ([commit](https://github.com/rails/rails/commit/5f777e4b5ee2e3e8e6fd0e2a208ec2a4d25a960d))
> >
> > -*   Added `#on_weekend?`, `#next_weekday`, `#prev_weekday` methods to `Date`,
> > +*   Added `#on_weekend?`, `#on_weekday?`, `#next_weekday`, `#prev_weekday` methods to `Date`,
> >      `Time`, and `DateTime`.
> >      ([Pull Request](https://github.com/rails/rails/pull/18335))
>
> We can add this PR link (23687) in the release notes too. The PR 18335
> doesn't add on_weekday.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/23687/files#r52901426>.
>


-- 
Vipul A.M.
+91-8149-204995

**Pull request merged**